### PR TITLE
171 xua simpleclient encoding problem on multiparted

### PR DESCRIPTION
**Pull request by ralych**
fix multiparted encoding bug

### CODE REVIEW

File: .github/workflows/java_ci.yml [hunk 1]
```diff
@@ -41,7 +41,8 @@ jobs:
         env:
           ARTF_USER: ${{ secrets.ARTF_USER }}
           ARTF_PASS: ${{ secrets.ARTF_PASS }}
-        run: mvn install javadoc:jar source:jar deploy --batch-mode --no-transfer-progress --file pom.xml -s $GITHUB_WORKSPACE/settings.xml
+        run: printenv
```
Comment (qligier) [inline .github/workflows/java_ci.yml:44]: This will leak the secrets

**Comment by qligier** [comment]
Also, we should not deploy snapshot versions on pull requests, nor do it in the middle of the java_ci

**Comment by ralych** [comment]
yes no deploy on PR, but it should only run on develop branch and master.
on develop we should have after merges deploys to snapshot repository. other wise you have always to build everything.
thats what we miss today.

File: .github/workflows/java_ci.yml (hunk 1 above)
Comment (ralych) [reply .github/workflows/java_ci.yml]: removed in last commit

**Review by qligier — APPROVED** [review]

**Pull request merged**